website/templates/emails/contributor_added.txt.mako

```diff
@@ -1,7 +1,13 @@
+<%!
+    from website import settings
+%>
+
 Hello ${user.fullname},
 
 ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the project "${node.title}" on the Open Science Framework: ${node.absolute_url}
 
+You will automatically be subscribed to notification emails for comments and updated files for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
```

Comment:
Should this also include a link to the project's settings (ditto other emails with this language)? I'm not certain it should; this may be more confusing for the user than its worth